src/ctl/src/cmd_impl/hummock/sst_dump.rs

```diff
@@ -55,18 +76,33 @@ pub async fn sst_dump() -> anyhow::Result<()> {
                 println!("Key Range: None");
             }
 
+            let data_path = sstable_store.get_sst_data_path(id);
             println!("Block Metadata:");
             for (i, block_meta) in sstable_meta.block_metas.iter().enumerate() {
+                // Retrieve encoded block data in bytes
+                let store = sstable_store.store();
+                let block_loc = BlockLocation {
+                    offset: block_meta.offset as usize,
+                    size: block_meta.len as usize,
+                };
+                let block_data = store.read(&data_path, Some(block_loc)).await?;
+
+                // Retrieve checksum and compression algorithm used from the encoded block data
+                let len = block_data.len();
+                let checksum = (&block_data[len - 8..]).get_u64_le();
```

Comment:
Is it possible to reuse the decode logic in risingwave storage crate? I remembered there is a Block::decode there.